index.html

```diff
@@ -0,0 +1,94 @@
+            </section>
+
+            <div class="tweets">
+                <img src="https://resizing.flixster.com/8_DH2ZydUioiOQWKQvJu4x-I8uY=/300x300/v2/https://resizing.flixster.com/v6Vr2guW06e5SIRw4eIyKgcgWH8=/ems.cHJkLWVtcy1hc3NldHMvbW92aWVzLzg2NDBkOWIyLTc1YzEtNGZjNS04MWY2LTRkYTEwYjA4NTRmZS5qcGc=">
+                <div class="name"><h4>Walid Wuhara <span>@WalidWuhara222   ·  18m</span></h4>
+                <p>What's up with hot dogs anyway?
+                </p></div>
+
+            </div>
+            <div class="tweets">
+                <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcS_nA12FrzHbsnzHQlTp3BXvE3Ou7iera-oJQ&usqp=CAU">
+                <div class="name"><h4>Grimes <span>@FlowerGirl12121   ·  2h</span></h4>
+                <p>I have an anime profile picture now.
+                </p></div>
+
+            </div>
+            <div class="tweets">
+                <img src="https://ibighit.com/bts/images/profile/proof/member/member-v.jpg">
+                <div class="name"><h4>K-Pop fan <span>@ElonMusk    ·  4h</span></h4>
+                <p>We need to end war to enjoy Gangnam Style. I will not rest.
+                </p></div>
+
+            </div>
+            <div class="tweets">
+                <img src="https://www.poornima.edu.in/wp-content/uploads/2022/06/Rajesh-e1654063757415.jpg">
+                <div class="name"><h4>Elon Musk Hater  <span>@ElonMuskHater69 ☑️  ·  18h</span></h4>
+                <p><a href="#">@ElonMusk</a> is very bad!!!! Money bad!! Capitalism bad!! But me pay to get blue checkmark.
+                </p></div>
+            </div>
+        </section>
+        <section class="right-side">
+            <div class="search-bar">
+                <input type="text" placeholder="Search 'Guacamole'">
+            </div>
+            <div class="whats-happening">
+                <h3>What's happening</h3>
+            </div>
+            <div class="who-to-follow">
+                <h3>Who to follow</h3>
+                <div class="accounts-to-follow">
+                <img src="https://www.poornima.edu.in/wp-content/uploads/2022/06/Rajesh-e1654063757415.jpg">
+                <div class="name"><h4 class="account-name">Elon Musk Hater </h4>
+                    <h4 class="username">@ElonMuskHater69 ☑️</h4>
+                </div>
+                </div>
+                <div class="accounts-to-follow">
+                    <img src="https://images.mktw.net/im-654236?width=1280&size=1">
+                    <div class="name"><h4 class="account-name">Elon Musk </h4>
+                        <h4 class="username">@ElonMusk ☑️</h4>
+                    </div>
+                    </div>
+                    <div class="accounts-to-follow">
+                        <img src="https://static01.nyt.com/images/2022/09/28/multimedia/00xp-shakira/00xp-shakira-mediumSquareAt3X.jpg">
+                        <div class="name"><h4 class="account-name">Shakira </h4>
```

Comment:
A lot of class creations which can be positive however I'm wondering if it could save you time in future by using selectors in your css and use methods like .name > h4 instead. Saying that; I think the readability is great!